### ISSUE TITLE
Worker died RtpStreamSend::Insert

### ISSUE THREAD

**Issue by oto313** [versatica/mediasoup#923]
### Your environment
- docker container - node:18-bullseye
- mediasoup version: 3.10.0

### Issue description
The worker died after couple of weeks of running. I do not have any more detail about it except following logs.

```
2022-10-11T06:56:02.387Z mediasoup:ERROR:Worker (stderr) (ABORT) RTC::RtpStreamSend::Insert() | failed assertion `this->buffer.size() <= MaxSeq': StorageItemBuffer contains more than 65535 entries
2022-10-11T06:56:02.633Z mediasoup:ERROR:Worker worker process died unexpectedly [pid:34, code:null, signal:SIGABRT]
```
Do you have any idea what could be a root cause? I could try to investigate it further, if I can reproduce it in a shorter time that 2 weeks.

Sorry it is rare crash, so I dont have crash dump.

**Comment by jmillan** [comment]
Thanks for the report. We'll look into it.

**Comment by ibc** [comment]
3.10.0 is kinda "old", latest version is 3.10.8 and ARAIR there were changes/fixed about this. @jmillan do you recall it? Changelog doesn't help much.

**Comment by oto313** [comment]
Ok, If you think there were some changes that might fix the bug, I can try to update and hope it wont happen. Thanks

**Comment by ibc** [comment]
> Ok, If you think there were some changes that might fix the bug, I can try to update and hope it wont happen. Thanks

Not 100% sure. Please try latest version and confirm.

**Comment by jmillan** [comment]
Indeed could have been fixed in 10.3.5 , here (10cfd3af19b0b025e3b581c63eb917c1dea14d81). 

```md

### 3.10.5

* `RtpStreamSend`: Do not store too old RTP packets (PR #885).
```

**Comment by jmillan** [comment]
Please @oto313, reopen this issue if you experience the problem with the newest version.
